e2e/chat.spec.ts

```diff
@@ -1,39 +1,127 @@
-import { test, expect } from '@playwright/test';
+import { describe, expect, beforeEach, getWalletId, test } from './fixtures';
 import { Chat } from './Chat';
+import { MetaMask } from './Metamask';
+import { EvmWalletID } from './Wallet';
 
-test('renders intro message', async ({ page }) => {
-  const chat = new Chat(page);
-  await chat.goto();
+describe('chat', () => {
+  test('renders intro message', async ({ page, context }) => {
+    const chat = new Chat(page);
+    await chat.goto();
 
-  expect(chat.messageContainer).not.toBeNull();
+    expect(chat.messageContainer).not.toBeNull();
 
-  await expect(page.getByText('Welcome to Oros!')).toBeVisible();
+    await expect(page.getByText("let's get started").first()).toBeVisible();
```

Comment:
`let's get started` appears twice - once as the header and then again in the paragraph that follows